packages/cosmic-swingset/lib/ag-solo/vats/vat-mints.js

```diff
@@ -1,37 +1,37 @@
 import harden from '@agoric/harden';
-import { makeMint } from '@agoric/ertp';
+import produceIssuer from '@agoric/ertp';
 
 import makeStore from '@agoric/store';
 
 // This vat contains two starting mints for demos: moolaMint and
 // simoleanMint.
 
 function build(_E, _log) {
-  const mints = makeStore();
+  const mintsAndMath = makeStore();
 
   const api = harden({
-    getAllAssetNames: () => mints.keys(),
-    getAssay: assetName => {
-      const mint = mints.get(assetName);
-      mint.getAssay();
+    getAllAssetNames: () => mintsAndMath.keys(),
+    getIssuer: assetName => {
+      const mint = mintsAndMath.get(assetName);
+      mint.getIssuer();
     },
-    getAssays: assetNames => assetNames.map(api.getAssay),
+    getAssays: assetNames => assetNames.map(api.getIssuer),
 
     // NOTE: having a reference to a mint object gives the ability to mint
     // new digital assets, a very powerful authority. This authority
     // should be closely held.
-    getMint: mints.get,
+    getMint: name => mintsAndMath.get[name].mint,
```

Comment:
We could change this to get the `IssuerResults`  (the object with properties like `mint`, `issuer` etc.) . I think this code as is won't work because it should be `.get(name)`